core/Pos/Binary/Crypto.hs

```diff
@@ -142,7 +142,7 @@ secretKeyLength, publicKeyLength, signatureLength, chainCodeLength,
     encryptedKeyLength, passphraseLength :: Int
 secretKeyLength = 32
 publicKeyLength = 32
-encryptedKeyLength = 96
+encryptedKeyLength = 128
```

Comment:
we use `encryptedKeyLength = 96` in `cardano-sl-0.5-rc`. This change was reverted here https://github.com/input-output-hk/cardano-sl/pull/1086